pre-reg-113-115-application-default.properties

```diff
@@ -0,0 +1,86 @@
+#helm chart properties
+#db.source.dbuser.password
+#db.destination.dbuser.password
+#server.port
+#mosip.base.url
+#mosip.prereg.client.secret
+#encryptBaseUrl
+#decryptBaseUrl
+#source.object.store.secretkey
+#destination.object.store.secretkey
+#isNewDatabase=true
+
+#Database config
+mosip.source.database.hostname=postgres-postgresql.postgres
+mosip.source.database.port=5432
+source.db.name=mosip_prereg_qa3
+# primary database connection
+datasource.primary.jdbcUrl=jdbc:postgresql://${mosip.source.database.hostname}:${mosip.source.database.port}/${source.db.name}
+datasource.primary.username=postgres
+datasource.primary.password=${db.source.dbuser.password}
+datasource.primary.driver-class-name=org.postgresql.Driver
+
+mosip.destination.database.hostname=postgres-postgresql.postgres
+mosip.destination.database.port=5432
+destination.db.name=mosip_prereg_qa_upgrade
+# secondary database connection
```

Comment:
In the comments, instead of primary and secondary, lets say source and destination so it is more understandable

Reply:
Done.